docs/README.md

```diff
@@ -1,29 +1,28 @@
 # Lockbook
 
 ## About
-_The best place to store and share thoughts_
+_The private, polished note-taking platform._
 
-Privacy shouldn’t be a compromise. That’s why we made Lockbook, a companion for recording thoughts on all your devices. Record, sync, and share your notes with apps engineered to feel like home on every platform. We collect no personal information and encrypt your notes so even _we_ can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
+Privacy shouldn't be a compromise. That's why we made Lockbook, a companion for recording thoughts on all your devices. Record, sync, and share your notes with apps engineered to feel like home on every platform. We collect no personal information and encrypt your notes so even _we_ can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
```

Comment:
what's happening here?

Reply:
Not sure, but the new one in the diff are apostrophes for sure. I made sure to type out that part when I noticed the diff.